src/content/en/tools/lighthouse/_toc.yaml

```diff
@@ -1,183 +1,7 @@
-  - title: Avoids Date.now()
-    path: /web/tools/lighthouse/audits/date-now
-  - title: Avoids Deprecated APIs
-    path: /web/tools/lighthouse/audits/deprecated-apis
-  - title: Avoids document.write()
-    path: /web/tools/lighthouse/audits/document-write
-  - title: Avoids Mutation Events In Its Own Scripts
-    path: /web/tools/lighthouse/audits/mutation-events
-  - title: Avoids Old CSS Flexbox
-    path: /web/tools/lighthouse/audits/old-flexbox
-  - title: Avoids Requesting The Geolocation Permission On Page Load
-    path: /web/tools/lighthouse/audits/geolocation-on-load
-  - title: Avoids Requesting The Notification Permission On Page Load
-    path: /web/tools/lighthouse/audits/notifications-on-load
-  - title: Displays Images With Incorrect Aspect Ratio
-    path: /web/tools/lighthouse/audits/aspect-ratio
-  - title: Includes Front-End JavaScript Libraries With Known Security Vulnerabilities
-    path: /web/tools/lighthouse/audits/vulnerabilities
-  - title: Manifest's short_Name Won't Be Truncated When Displayed On Homescreen
-    path: /web/tools/lighthouse/audits/manifest-short_name-is-not-truncated
-  - title: Links to cross-origin destinations are unsafe
-    path: /web/tools/lighthouse/audits/noopener
-  - title: Prevents Users From Pasting Into Password Fields
-    path: /web/tools/lighthouse/audits/password-pasting
-  - title: Some Insecure Resources Can Be Upgraded To HTTPS
-    path: /web/tools/lighthouse/audits/mixed-content
-  - title: Uses HTTPS
-    path: /web/tools/lighthouse/audits/https
-  - title: Uses HTTP/2 For Its Own Resources
-    path: /web/tools/lighthouse/audits/http2
-  - title: Uses Passive Event Listeners To Improve Scrolling Performance
-    path: /web/tools/lighthouse/audits/passive-event-listeners
-  - title: Avoids Web SQL
-    path: /web/tools/lighthouse/audits/web-sql
-    status: deprecated
-- title: SEO
-  section:
-  - title: Document Does Not Have A Meta Description
-    path: /web/tools/lighthouse/audits/description
-  - title: Document Doesn't Have A Title Element
-    path: /web/tools/lighthouse/audits/title
-  - title: Document doesn't have a valid hreflang
-    path: /web/tools/lighthouse/audits/hreflang
-  - title: Document doesn't have a valid rel=canonical
-    path: /web/tools/lighthouse/audits/canonical
-  - title: Document Doesn't Use Legible Font Sizes
-    path: /web/tools/lighthouse/audits/font-sizes
-  - title: Document uses plugins
-    path: /web/tools/lighthouse/audits/plugins
-  - title: Links Do Not Have Descriptive Text
-    path: /web/tools/lighthouse/audits/descriptive-link-text
-  - title: Page has unsuccessful HTTP status code
-    path: /web/tools/lighthouse/audits/successful-http-code
-  - title: Page is blocked from indexing
-    path: /web/tools/lighthouse/audits/indexing
-  - title: robots.txt is not valid
-    path: /web/tools/lighthouse/audits/robots
-  - title: Tap targets are not sized appropriately
-    path: /web/tools/lighthouse/audits/tap-targets
```

Comment:
You've removed this from the TOC, but left the content. That means it will still be online and people can get to it via existing links, search, etc. There's no indication on the page that it now lives on web.dev, or that it may be out of date. And, because it's not in the TOC, it won't show up in the left hand nav, and provide no way for a user to navigate away from it.

Reply:
for posterity: kayce fixed this and the following comments in https://github.com/google/WebFundamentals/pull/8691